docs/layouts/shortcodes/packageref.html

```diff
@@ -3,7 +3,7 @@
 {{/* - unscopedPackageName: The *unscoped* name of the package being linked to. */}}
 
 {{- $unscopedPackageName := .Get 0 -}}
-{{- $path := delimit (slice "/docs/api/v1/" $unscopedPackageName ".md" ) "" -}}
+{{- $path := delimit (slice "/docs/apis/" $unscopedPackageName ".md" ) "" -}}
```

Comment:
Effectively the same comment as I left [here](https://github.com/microsoft/FluidFramework/pull/19093/files#r1442295361). Probably a separate task, but this template should take in a version string and link to the API docs for that version.

Reply:
Same with `apiref.html`

Reply:
I thought our server redirect policy will point all /apis/ to latest version, right?

Reply:
You're right. And actually, we probably won't have many cases where we want to specifically point to older versions of API docs. It may still come up, so we may want to add the ability to do this, but I guess it isn't urgent.